src/main/kotlin/lotto/model/Lottos.kt

```diff
@@ -0,0 +1,38 @@
+package lotto.model
+
+/**
+ * 로또 당첨번호 및 통계 결과 클래스
+ * */
+class Lottos private constructor(private val price: Price, private val purchasedLotto: List<Lotto>, private val lastWinNumber: Lotto) {
+    fun compareLottoList(): LottoStatisticFormat {
```

Comment:
```suggestion
class Lottos private constructor(private val price: Price, private val purchasedLotto: List<Lotto>, private val lastWinNumber: Lotto) {

    fun compareLottoList(): LottoStatisticFormat {
```
클래스명과 함수 정의 도입부 사이에는 한 줄을 띄워주는 것이 좋습니다